src/test/unit_tests/test_notifications.js

```diff
@@ -0,0 +1,203 @@
+/* Copyright (C) 2016 NooBaa */
```

Comment:
You can update the year
```suggestion
/* Copyright (C) 2025 NooBaa */
```